django_extensions/management/modelviz.py

```diff
@@ -148,7 +151,7 @@ def add_relation(self, field, model, extras=""):
             related_query_name = field.related_query_name()
             if self.verbose_names and related_query_name.islower():
                 related_query_name = related_query_name.replace('_', ' ').capitalize()
-            label = '{} ({})'.format(label, force_str(related_query_name))
+            label = '{} ({})'.format(label, force_text(related_query_name))
```

Comment:
This line is breaking Python 2.
In the beginning of the file there is commented line `from __future__ import unicode_literals`. Then in Python 2 normal strings are not unicode. When non unicode string is formatted with `format` and there are unicode string as arguments it will rasie `UnicodeEncodeError`.

Reply:
Could you maybe make a PR with a test for this ?

Reply:
I'm tried to reproduce it on my local machine (because it was just guessing based on reading the code) and I found that it can happen only if all of the following are met:

- running on Python 2
- there are `verbose_name` of ForeignKey/ManyToManyFIeld set which contains non ASCII characters
- `graph_models` is executed with `--verbose-names` argument

Did you have any suggestions about the test case what exactly to test?

Reply:
I created #1247 with test and fix for the issue. You can look at it.